tests/spread/integration/bc/task.yaml

```diff
@@ -0,0 +1,12 @@
+summary: Integration tests for bc
+
+execute: |
+  # Chisel a minimum number of slices to give us a runnable system that we can
+  # test in.
+  rootfs="$(install-slices bc_bins)"
+
+  chroot "${rootfs}/" bc -h
+  if [ "${x}" -ne 3 ]; then
```

Comment:
Oh, I just noticed, there is a line missing.
Can you change this whole section to the following?
```
  chroot "${rootfs}/" bc -h
  x=$(echo "1+2" | chroot "${rootfs}/" bc)
  if [ "${x}" -ne 3 ]; then
    echo "bc failed 1+2"
    exit 1
  fi
```